x-pack/platform/plugins/shared/onechat/public/application/components/tools/form/registry/tool_types/workflow.tsx

```diff
@@ -31,6 +31,7 @@ export const workflowToolRegistryEntry: ToolTypeRegistryEntry<WorkflowToolFormDa
     ...commonToolFormDefaultValues,
     type: ToolType.workflow,
     workflow_id: '',
+    exclude_details: false,
```

Comment:
makes sense to enable by default?

Reply:
Decision: it shouldn't be configurable, and the new behavior should be to always exclude details.